reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/user_data_master.yml

```diff
@@ -21,13 +21,14 @@ mounts:
 - [ /dev/xvdc, /var/lib/etcd, xfs, "defaults" ]
 - [ /dev/xvdd, /var/lib/origin/openshift.local.volumes, xfs, "defaults,gquota" ]
 
-
 write_files:
 - content: |
     DEVS='/dev/xvdb'
     VG=docker_vol
     DATA_SIZE=95%VG
-    EXTRA_DOCKER_STORAGE_OPTIONS="--storage-opt dm.basesize=3G"
+    STORAGE_DRIVER=overlay2
```

Comment:
I'm afraid that this is not enough. `docker-storage-setup` won't create fs and mount it as `/var/lib/docker`, it has to be done manually :-( (at least AFAIK)